testsuite/forge-cli/src/suites/realistic_environment.rs

```diff
@@ -238,7 +238,7 @@ pub(crate) fn realistic_env_graceful_overload(duration: Duration) -> ForgeConfig
         .with_initial_fullnode_count(20)
         .add_network_test(wrap_with_realistic_env(num_validators, TwoTrafficsTest {
             inner_traffic: EmitJobRequest::default()
-                .mode(EmitJobMode::ConstTps { tps: 15000 })
+                .mode(EmitJobMode::ConstTps { tps: 30000 })
```

Comment:
As the TPS is increased, should we be updating the success criteria for the test as well?
Is the test passing with the current success critieria?

Reply:
The test was passing "too well" -- almost at 14K TPS. Now that we've made it overload, it is at 11K TPS. To avoid test failure noise, I'll change the success criteria after we see a few runs though